.rhcicd/clowdapp-backend.yaml

```diff
@@ -142,6 +138,18 @@ objects:
           value: ${INTERNAL_RBAC_ENABLED}
         - name: INTERNAL_ADMIN_ROLE
           value: ${INTERNAL_RBAC_ADMIN}
+        - name: INVENTORY_API_AUTHN_CLIENT_ID
+          valueFrom:
+            secretKeyRef:
+              name: kessel-authentication
+              key: inventory.client.id
```

Comment:
`inventory.client.id` / `relations-api.client.id`

Could make this consistent? Either `-api` in both or in none.

Reply:
Yes, changed!